src/Misc/MiddleWare.cpp

```diff
@@ -266,6 +266,38 @@ static void idsFromMsg(const char* msg, int* part, int* kit, int* voice)
     }
 }
 
+/**
+ * Class responsible for sending chained requests of "/wavetable-rt-params"
+ */
+class WaveTableRequestHandler
+{
+    bool waitForAdPars[NUM_MIDI_PARTS][NUM_KIT_ITEMS][NUM_VOICES];
+
+public:
+    WaveTableRequestHandler()
+    {
+        for(std::size_t p = 0; p < NUM_MIDI_PARTS; ++p)
```

Comment:
Unusual indentation

Reply:
I know, but if the for loops are indented, it looks worse to me. Right now, you can easily see that the 4 variables are `p, k, v, i` and what their loop range is. I'd prefer to leave it like this.

Reply:
Sure thing. :+1: